qemu/tests/cfg/block_resize.cfg

```diff
@@ -27,6 +34,21 @@
                 blk_extra_params_stg = "wwn=0x5000123456789abc"
             Windows:
                 need_rescan = yes
+            variants:
+                - default_partition:
+                - with_gpt_ext4:
+                    only Linux
+                    labeltype = gpt
+                    disk_change_ratio = "${extend_ratio}"
+                - with_msdos_xfs:
+                    only Linux
+                    fstype = xfs
+                    disk_change_ratio = "${extend_ratio}"
+                - with_gpt_xfs:
+                    only Linux
+                    labeltype = gpt
+                    fstype = xfs
+                    disk_change_ratio = "${extend_ratio}"
```

Comment:
better add a new case variants for with_md5_check, currently config will impact case with_iozone test, it doesn't need do md5 check after iozone test original, if added md5 check step, when md5 check failed then all with_iozone test cases will failed.

Reply:
> better add a new case variants for with_md5_check, currently config will impact case with_iozone test, it doesn't need do md5 check after iozone test original, if added md5 check step, when md5 check failed then all with_iozone test cases will failed.

We have 16 cases for block resize, It will create new 16 test cases just adding md5 checking. is it worth to do?   We may consider merge them as common checking after disk resizing.

Reply:
> > better add a new case variants for with_md5_check, currently config will impact case with_iozone test, it doesn't need do md5 check after iozone test original, if added md5 check step, when md5 check failed then all with_iozone test cases will failed.
> 
> We have 16 cases for block resize, It will create new 16 test cases just adding md5 checking. is it worth to do? We may consider merge them as common checking after disk resizing.

We may put md5 checking before iozone testing.

Reply:
If iozone and md5 check are not the same cases, we can separate them with flag like "check_md5 = yes" . If izone and md5 is important for each case, we can merge these cases to one.